doc/source/whatsnew/v0.18.1.txt

```diff
@@ -18,8 +18,7 @@ Highlights include:
 New features
 ~~~~~~~~~~~~
 
-
-
+- ``pd.crosstab()`` has gained ``normalize`` argument for normalizing frequency tables (:issue:`12569`). Examples in updated docs :ref:`here <reshaping.crosstabulations>`.
```

Comment:
@jreback. Thanks. I'm going to be traveling away from my computer for a
little over two weeks. I'm not sure what the timing is for 0.18.1. I'm
happy to make changes when I get back, but won't be able to do anything
till then.
On Mon, Apr 4, 2016 at 10:59 AM Jeff Reback <notifications@github.com>
wrote:

> In doc/source/whatsnew/v0.18.1.txt
> <https://github.com/pydata/pandas/pull/12578#discussion_r58418709>:
>
> > @@ -18,8 +18,7 @@ Highlights include:
> >  New features
> >  ~~~~~~~~~~~~
> >
> > -
> > -
> > +- ``pd.crosstab()`` has gained ``normalize`` argument for normalizing frequency tables (:issue:`12569`). Examples in updated docs :ref:`here <reshaping.crosstabulations>`.
>
> why don't you add a mini-example here (same one)
>
> —
> You are receiving this because you were mentioned.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/12578/files/39718b8f4f398d9cdf6c38583453c804675a5e6a#r58418709>
>